app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
I'm thinking no, that we'd just run it weekly rather than at every sync.  It'd likely solve the race condition for other units as well.  I don't think its a problem to run weekly, as its done that way on the satellite.  The user could always add a new cronjob if they wanted it daily, or more often.